kafka/source/pkg/apis/sources/v1beta1/kafka_types.go

```diff
@@ -77,6 +77,10 @@ type KafkaSourceSpec struct {
 	// +optional
 	ConsumerGroup string `json:"consumerGroup,omitempty"`
 
+	// AvroSchema is the AVRO schema used to decode the message.
+	// +optional
+	AvroSchema string `json:"avroSchema,omitempty"`
```

Comment:
As the comment https://github.com/knative/eventing-contrib/pull/1393#issuecomment-661044687, let's keep this on hold

Reply:
this is not the schema uri, this is the schema itself.  This is as per this [comment](https://github.com/knative/eventing-contrib/pull/1393#issuecomment-664428863).